pkg/operator/components/rbac.go

```diff
@@ -98,8 +98,8 @@ var (
 				},
 				Resources: []string{
 					"machinedisruptionbudgets",
-					"machinedisruptionbudgets/status",
```

Comment:
why do you remove this rule?

Reply:
I thought it was a TYPO, and there should be only `machinehealthchecks/status`. I'll add it back.